packages/stream_chat_flutter/lib/src/message_input/audio_message_send_button/stream_audio_message_send_button.dart

```diff
@@ -0,0 +1,224 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:stream_chat_flutter/src/message_input/audio_message_send_button/stream_audio_message_controllers.dart';
+import 'package:stream_chat_flutter/src/message_input/audio_message_send_button/stream_audio_message_gesture_state.dart';
+import 'package:stream_chat_flutter/src/message_input/audio_message_send_button/stream_audio_message_overlays.dart';
```

Comment:
Let's create a single export file for the audio_message_send_button folder, and maybe even export it from stream_chat_flutter?

Reply:
I wasn't sure if I had to create a rollup file, since I haven't seen any on the source. I would be happy to do it though 🙂